packages/flutter_reactive_ble/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: flutter_reactive_ble
 description: Reactive Bluetooth Low Energy (BLE) plugin that can communicate with multiple devices
-version: 5.1.1
+version: 5.2.0
```

Comment:
Should we make any changes to any Podspec files?

Reply:
Podspec.lock were removed for library packages

Reply:
Well, the only one out there doesn't seem to have ever been changed. It's probably okay as is.

(packages/reactive_ble_mobile/ios/reactive_ble_mobile.podspec)